README.md

```diff
@@ -288,9 +291,29 @@ make test_unit
 make test_e2e_shannon_relay
 ```
 
-## 8. Troubleshooting
+## 8. Running Localnet
 
-### 8.1. Docker Permissions Issues - Need to run sudo?
+You can use path configuration under `/local` to spin up a local development environment using `Kind` + `Tilt`.
```

Comment:
See what we have in [1]

Once the user has gotten to step `8`, they don't want to relearn pre-requisites again, they just want to do that all upfront.

https://dev.poktroll.com/develop/developer_guide/quickstart.

Reply:
I've moved the pre-requisites to 1.1